src/codec/parameters.rs

```diff
@@ -43,6 +45,60 @@ impl Parameters {
     pub fn id(&self) -> Id {
         unsafe { Id::from((*self.as_ptr()).codec_id) }
     }
+
+    #[inline]
+    pub fn set_width(&mut self, value: u32) {
```

Comment:
Should `value` be an `i32` for `set_(width|height)` so that it is safer to cast to `c_int` ?

Reply:
The `set_width` and `set_height` calls that I took from the encoder does it with u32. I think both cases are fine, since if you specify i32, the user can put in a negative width/height which doesn't make sense either. I'll leave as it to stay consistent with the rest of the wrapper.

Reply:
Ok makes sense. I've opened an issue for this as I think we could improve the code by patching all these u32 -> c_int into i32. Issue : https://github.com/flavioroth/ffmpeg-rs/issues/21